helm-charts/FATE/templates/hdfs-nn.yaml

```diff
@@ -168,6 +168,9 @@ spec:
       port: 9000
       targetPort: 9000
       protocol: TCP
+      {{- if eq .Values.modules.hdfs.namenode.type "NodePort" "LoadBalancer" }}
```

Comment:
why mixing `nodeport` and `loadbalancer` here?

Reply:
We can deploy a hdfs cluster, the entrance use 'NodePort' or 'LoadBalancer'.